Example/ConsentViewController_ExampleTests/MessageWebViewControllerSpec.swift

```diff
@@ -120,7 +120,7 @@ class MessageWebViewControllerSpec: QuickSpec, GDPRConsentDelegate, WKNavigation
                         ])
 
                         messageWebViewController.userContentController(userContentController, didReceive: message)
-                        let expectedPMURL = "https://notice.sp-prod.net/privacy-manager/index.html?site_id=1&consentUUID=uuid&message_id=122058"
```

Comment:
@vilasma I think we could change this spec. The `WebView` doesn't "care" if the URL is `notice.sp-prod` or the CDN one. All we should test is that the query param `message_id` contains the right value (PM ID)

Reply:
@andresilveirah updated the spec.